cryptography/Cargo.toml

```diff
@@ -41,6 +42,9 @@ zeroize = { workspace = true, features = ["zeroize_derive"] }
 version = "0.2.15"
 features = ["js"]
 
+[target.'cfg(unix)'.dependencies]
+libc = "0.2"
```

Comment:
can we do a full version specification?

Reply:
I guess this should also go at the root `Cargo.toml`.